passvault/passvault.go

```diff
@@ -364,8 +365,16 @@ func (records *Records) AddNewRecord(name, password string, admin bool, userType
 }
 
 // ChangePassword changes the password for a given user.
-func (records *Records) ChangePassword(name, password, newPassword string) (err error) {
+func (records *Records) ChangePassword(name, password, newPassword, hipchatName string) (err error) {
 	pr, ok := records.GetRecord(name)
+
+	if len(hipchatName) != 0 {
```

Comment:
It might make sense to rename this API endpoint if it's going to be expanding beyond just changing the password